op-acceptance-tests/acceptance-tests.yaml

```diff
@@ -82,5 +82,5 @@ gates:
   - id: sync-test-op-node
     description: "Sync tests for op-node with external networks via the op-sync-tester - tests run daily."
     tests:
-      - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/sync_tester_ext_el
+      - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/sync_tester/sync_tester_ext_el
```

Comment:
Would be nice to check this change by manually triggering the nightly CI via CircleCI Console.